frontend/src/pages/auth/components/forgot-password-form/forgot-password-form.tsx

```diff
@@ -0,0 +1,62 @@
+import { Input, Link, Notification } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/app-route.enum';
+import { useAppForm, useCallback } from '~/libs/hooks/hooks.js';
+import {
+  type AuthRequestPasswordDto,
+  requestPasswordValidationSchema,
+} from '~/packages/auth/auth.js';
+
+import { AuthSubmitButton } from '../auth-submit-button/auth-submit-button.js';
+import { DEFAULT_PAYLOAD } from './libs/constants/constants.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  onSubmit: (payload: AuthRequestPasswordDto) => void;
+};
+
+const ForgotPasswordForm: React.FC<Properties> = ({ onSubmit }) => {
+  const { control, errors, handleSubmit } = useAppForm({
+    defaultValues: DEFAULT_PAYLOAD,
+    validationSchema: requestPasswordValidationSchema,
+  });
+  const handleFormSubmit = useCallback(
+    (event_: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event_);
+    },
+    [handleSubmit, onSubmit],
+  );
+
+  return (
+    <>
+      <Notification />
```

Comment:
Why don't we render this component in `frontend\src\index.tsx`? It will be a lot easier to manage. We might forget to render it, if we use it like that 🤔

Reply:
I don't know. Maybe in some cases you will want to display notifications in different ways ( position , styling and so on ) so having app-wide notification component maybe is not really what we want. I just tested it -  notification in index.ts overrides any local styling.